lib/khmer.hh

```diff
@@ -154,6 +161,11 @@ class Kmer;
 typedef std::queue<Kmer> KmerQueue;
 typedef std::set<Kmer> KmerSet;
 
+// A function which takes a Kmer and returns true if it
+// is to be filtered / ignored
+typedef std::function<bool (Kmer&)> KmerFilter;
+typedef std::list<KmerFilter> KmerFilterList;
```

Comment:
Why `std::list`? This should be a `vector` unless there is a good reason to use `list` which is nearly never a good idea ;)

If there is a reason for `list` it is worth leaving a comment here to inform archeologists from the future who will ask the same question.